content/getting-started/device-farm-tutorial.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Use Alexa App on Device Farm (optional)"
+date: 2019-09-26T13:29:22+08:00
+draft: false
+chapter: false
+---
+
+If you can not download or use the Alexa App due to network or region, you can use [AWS Device Farm](https://aws.amazon.com/device-farm/) to start a real, physical phone hosted by AWS, onto which you can install and use the Alexa App. You can interact with this device by using a remote access session.  
+
+{{% notice info %}}
+In this section we will show you how to add Google account and install Alexa App on a remote Android device using AWS Device Farm. You can also start an IOS device, but you can not use Apple Store to install Apps or add your Apple ID to this device due to security reasons. You can only use the upload option to upload and install your apps. So we can not install Alexa app on IOS devices.
+{{% /notice %}}
+
+{{% notice info %}}
+AWS Device Farm comes with a one-time free trial of 1000 device minutes. After that you are charged $0.17 per device minute. For more information about pricing, see [AWS Device Farm Pricing](https://aws.amazon.com/device-farm/pricing/). 
+{{% /notice %}}
+
+## Start Device Farm
+1. Go to [AWS Device Farm](https://us-west-2.console.aws.amazon.com/devicefarm/home?region=us-east-1#/projects).
```

Comment:
the entire workshop use us-east-1 region.

Reply:
Device farm is global and not associated with a region.